jmix-flowui/flowui-themes/src/main/resources/META-INF/resources/themes/jmix-lumo/component/jmix-horizontal-menu.css

```diff
@@ -14,41 +14,54 @@
  * limitations under the License.
  */
 
-.jmix-navigation-menu-bar {
+.jmix-horizontal-menu {
     color: var(--lumo-secondary-text-color);
 }
 
-.jmix-navigation-menu-bar-root-item:hover {
+.jmix-horizontal-menu-root-item:hover {
   color: var(--lumo-contrast-80pct);
 }
 
-.jmix-navigation-menu-bar vaadin-menu-bar-button {
+.jmix-horizontal-menu vaadin-menu-bar-button {
   color: inherit;
   background-color: transparent;
   border-radius: var(--lumo-border-radius-m);
 }
 
-.jmix-navigation-menu-bar vaadin-menu-bar-button[expanded] {
+.jmix-horizontal-menu vaadin-menu-bar-button[expanded] {
   background-color: var(--lumo-primary-color-10pct);
 }
 
-vaadin-menu-bar-item .jmix-navigation-menu-bar-menu-item,
-vaadin-menu-bar-list-box .jmix-navigation-menu-bar-menu-item {
+.jmix-horizontal-menu-content-item {
+  gap: var(--lumo-space-s);
+  align-items: center;
+}
+
+.jmix-horizontal-menu-content-item vaadin-icon.prefix-component {
+  width: var(--lumo-icon-size-s);
+  height: var(--lumo-icon-size-s);
+}
+
+vaadin-menu-bar-list-box .jmix-horizontal-menu-content-item .text-component:not(:last-child) {
+  margin-right: var(--lumo-space-l);
```

Comment:
you can't use left/right variants, because they are not adapted to **rtl** mode. Use `inline-start/end` instead.